ngxunprivinst.sh

```diff
@@ -328,8 +328,12 @@ extract() {
             echo "Use command \"ldd $ABSPATH/usr/sbin/nginx\" to check unmet dependencies." && \
             exit 1
         fi
+        TARGETVER=$($ABSPATH/usr/sbin/nginx -v 2>&1 | cut -d '(' -f 2 | cut -d ')' -f 1 | cut -d'-' -f 3 | tr -d 'r')
```

Comment:
The command using `cut` and `tr` is less readable in my opinion. `awk` is often more readable and clearer in terms of intent. However, `awk` is a separate utility compared to `cut` and `tr` as I understand it.

Reply:
Yep, the reason we don't use awk is it might not be available when cut/tr typically are.